package.json

```diff
@@ -34,60 +32,30 @@
     "@cfpb/cfpb-tables": "0.14.1",
     "@cfpb/cfpb-typography": "0.14.1",
     "@cfpb/rental-assistance-finder": "0.4.0",
+    "@frappe/esbuild-plugin-postcss2": "0.1.3",
```

Comment:
Can this go in devDependencies?

Reply:
Oh right we need all the build things in `dependencies` because of how we build things outside of the local environment, right?

Reply:
Yeah, the dep/devdep binary is kinda odd for a user-facing application.